src/OnboardingSPA/components/Drawer/DrawerPanel/DesignThemeStylesPreview.js

```diff
@@ -112,17 +118,23 @@ const DesignThemeStylesPreview = () => {
 	return (
 		<div className="theme-styles-preview--drawer">
 			<div className="theme-styles-preview--drawer__list">
+				{!globalStyles
+					&& <LivePreviewSkeleton count={Math.floor(THEME_VARIATIONS)}
+						className={'theme-styles-preview--drawer__list__item'} 
+						viewportWidth={900} skeletonLoadingTime={4000}/>}
```

Comment:
```suggestion
				{!globalStyles
					? <LivePreviewSkeleton count={Math.floor(THEME_VARIATIONS)}
						className={'theme-styles-preview--drawer__list__item'} 
						viewportWidth={900} skeletonLoadingTime={4000}/> : buildPreviews().slice( 0, MAX_PREVIEWS_PER_ROW )}
```

Reply:
Yes I did try this but apparently it does not rerender in this case and the Live Preview renders as an empty Box